tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "outDir": "lib",
     "rootDir": "src",
     "jsx": "react",
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "noUnusedLocals": false
```

Comment:
Why do we need this disabled?

Reply:
Nope, I accidentally pushed that. Will revert